src/web/src/components/Search/Searches.jsx

```diff
@@ -248,6 +248,19 @@ const Searches = ({ server } = {}) => {
           ref={inputRef}
           size="big"
         />
+
+        <Button
+          className="px-2"
+          disabled={!Object.keys(searches).length}
+          onClick={() => {
+            for (const search of Object.values(searches)) {
```

Comment:
i think this logic is probably fine, but would prefer to see it extracted to a `removeAll` function